.github/labeler.yml

```diff
@@ -0,0 +1,130 @@
+Automation:
+- changed-files:
+  - any-glob-to-any-file:
+    - .github/**
+
+CI:
+- changed-files:
+  - any-glob-to-any-file:
+    - ci/**
+
+dependencies:
+- changed-files:
+  - any-glob-to-any-file:
+    - ci/requirements/*
+
+topic-arrays:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/core/duck_array_ops.py
+
+topic-backends:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/backends/**
+
+topic-cftime:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/coding/*time*
+
+topic-CF conventions:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/conventions.py
+
+topic-combine:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/core/combine.py
+
+topic-dask:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/core/dask*
+    - xarray/core/parallel.py
+
+topic-DataTree:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/core/datatree*
+
+# topic-documentation:
+# - changed-files:
+#   - any-glob-to-any-file:
+#     - ['doc/*', '!doc/whats-new.rst', 'doc/**/*']
+
+topic-faq:
+- changed-files:
+  - any-glob-to-any-file:
+    - doc/howdoi.rst
+
+topic-groupby:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/core/groupby.py
+
+topic-html-repr:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/core/formatting_html.py
+
+topic-hypothesis:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/properties/*
+    - xarray/testing/strategies/*
+
+topic-indexing:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/core/indexes.py
+    - xarray/core/indexing.py
+
+topic-NamedArray:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/namedarray/*
+
+topic-performance:
+- changed-files:
+  - any-glob-to-any-file:
+    - asv_bench/benchmarks/**
+
+topic-plotting:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/plot/*
+    - xarray/plot/**/*
+
+topic-rolling:
+- changed-files:
+  - any-glob-to-any-file:
+    - xarray/core/rolling.py
+    - xarray/core/rolling_exp.py
```

Comment:
Should be `computation` not `core`

I'll leave the rest of the validating label stuff and pushing the updates up to you maintainers :)